sound/soc/sof/ipc4.c

```diff
@@ -97,6 +97,7 @@ EXPORT_SYMBOL(sof_ipc4_process_reply);
 static int ipc4_tx_wait_done(struct snd_sof_ipc *ipc, struct snd_sof_ipc_msg *msg,
 			     void *reply_data)
 {
+	struct sof_ipc4_msg *ipc4_msg = msg->msg_data;
```

Comment:
typo in commit message "ip4" and this should be a fixup no?

Reply:
Hm, I think it can not be a fixup as there might have been too many changes in ipc4 since it's introduction.